index.js

```diff
@@ -0,0 +1,78 @@
+const lengthCheck = (password) => {
+  if (password.length >= 8) {
+    return true
+  } else return false
+}
```

Comment:
An example of refactoring:
const lengthCheck = (password) => {
  return password.length >= 8
}